hermes/probe/probe.go

```diff
@@ -149,22 +140,24 @@ func (mp *Probe) runProbe(ctx context.Context, metricChan chan<- *cpmetrics.Even
 	var wg sync.WaitGroup
 	for _, t := range mp.targets {
 		wg.Add(1)
-		go func(t *Target) {
+		go func(t *target.Target) {
 			defer wg.Done()
-			var err error
-			if t.LatencyMetrics, err = metrics.NewMetrics(mp.config, t.Target); err != nil {
-				mp.logger.Errorf(err.Error())
-				return
+			if t.LatencyMetrics == nil {
+				var err error
```

Comment:
This isn't wrong, but here's an alternative that doesn't require pre-declaring `err`:

```
ms, err := metrics.NewMetrics(...)
if err != nil {
   ...
}
t.LatencyMetrics = ms
return
```

Reply:
Similarly, this seems neater to me, so I've changed it accordingly.